tests/framework/client/persistentvolume.go

```diff
@@ -0,0 +1,93 @@
+package client
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+	corev1 "k8s.io/client-go/applyconfigurations/core/v1"
+	v12 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+var createdPVs = make([]pvInfo, 0)
```

Comment:
The idea was to keep each "tracked" information in its own context, but if you think that keep it all in the client will increase the readability, or for some other reasons, I have no problem to change it :)

Reply:
I think this is fine as far we don't use client concurently, right?

Reply:
The idea is not to use another `client` concurrently because otherwise the `testClient` is bypassed. This has to be forethought of the developers and the reviewers. On the other hand, also if you use the `kubectl` or the `RestClient` you can bypass it.